specs/lib/mtlSpec.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+var path = require('path');
+var loadMtl = require('../../lib/mtl.js');
+
+var complexMaterialUrl = 'specs/data/box-complex-material/box-complex-material.mtl';
+var multipleMaterialsUrl = 'specs/data/box-multiple-materials/box-multiple-materials.mtl';
+var invalidMaterialUrl = 'invalid.mtl';
+
+function getImagePath(objPath, relativePath) {
+    return path.normalize(path.join(path.dirname(objPath), relativePath));
+}
+
+describe('mtl', function() {
```

Comment:
You can write up a separate issue for this, but are no tests here for how we handle invalid mtl files and we don't do any sort of parameter validation either.  Please write up an issue to review all functions for error handling and parameter validation.